src/com/mambu/apisdk/services/OwnedEntityService.java

```diff
@@ -0,0 +1,189 @@
+	 *            Mambu entity for which the owned entity is updated. Example, Client, Branch
+	 * @param parentEntityId
+	 *            entity id or encoded key for the parent
+	 * @param ownedEntityId
+	 *            the encoded key or id of the owned entity to be updated
+	 * @param ownedEntity
+	 *            the new owned entity object
+	 * @return updated owned entity
+	 * @throws MambuApiException
+	 */
+	public <T> boolean updateOwnedEntity(MambuEntity parentEntity, String parentEntityId, String ownedEntityId,
+			T ownedEntity) throws MambuApiException {
+
+		// Execute request for PATCH API to update custom field value for a Loan Account. See MBU-6661
+		// e.g. PATCH "{ "value": "10" }" /host/api/loans/accointId/custominformation/customFieldId
+
+		// Or API request to PATCH a linked field for the custom field value (see MBU-8514)
+		// PATCH '{ "linkedEntityKeyValue": "40288a13...." }'// /api/loans/abc123/custominformation/customFieldId
+
+		// Validate API is used for Supported entity
+		validateParentEntity(parentEntity);
+
+		// Submit API request
+		return serviceExecutor.patchOwnedEntity(parentEntity, parentEntityId, ownedEntity, ownedEntityId);
+
+	}
+
+	/***
+	 * Delete owned entity for a Mambu parent entity
+	 * 
+	 * @param parentEntity
+	 *            Mambu entity for which owned entity is deleted. Example, Client, Branch
+	 * @param parentEntityId
+	 *            entity id or encoded key for the parent
+	 * @param ownedEntityId
+	 *            the encoded key or id of the owned entity to be deleted
+	 * @return true if successful
+	 * @throws MambuApiException
+	 */
+	public boolean deleteOwnedEntity(MambuEntity parentEntity, String parentEntityId, String ownedEntityId)
+			throws MambuApiException {
+		// Execute request for DELETE API to delete custom field value for a client
+		// e.g. DELETE /host/api/clients/clientId/custominformation/customFieldId
+
+		validateParentEntity(parentEntity);
+
+		// Derived class will provide the owned entity managed by it. Example, Comment, CustomFieldValue
+		MambuEntity ownedEntity = getOwnedEntity();
+		return serviceExecutor.deleteOwnedEntity(parentEntity, parentEntityId, ownedEntity, ownedEntityId);
+
+	}
+
+	/**
+	 * Validate parent entity. This methods validates if the provided parent entity is supported by the extending class
+	 * 
+	 * @param parentEntity
+	 *            parent entity
+	 */
+	protected void validateParentEntity(MambuEntity parentEntity) {
+		Set<MambuEntity> supportedEntities = getSupportedEntities();
+
+		if (supportedEntities == null || !supportedEntities.contains(parentEntity)) {
+			throw new IllegalArgumentException("Parent Entity  " + parentEntity + " is not supported by "
+					+ getOwnedEntity());
+		}
+	}
+
+	/**
+	 * Validate API supported entity. This methods validates if the provided request is supported by the extending class
+	 * 
+	 * @param parentEntity
+	 *            parent entity
+	 */
+	protected void validateAPiIsSupported(Method method) {
```

Comment:
Please rename to reflect that API method or HTTP method is supported and also update method comment accordingly.